### ISSUE TITLE
Flaking test - ci-cos-cgroupv2-containerd-node-e2e

### ISSUE THREAD

**Issue by bobbypage** [kubernetes/kubernetes#105301]
#### Which jobs are flaking:

ci-cos-cgroupv2-containerd-node-e2e

#### Which test(s) are flaking:

k8s.io/kubernetes/test/e2e_node/summary_test.go

#### Testgrid link:

https://testgrid.k8s.io/sig-node-containerd#cos-cgroupv2-containerd-node-e2e

#### Reason for failure:

```
_output/local/go/src/k8s.io/kubernetes/test/e2e_node/summary_test.go:54
Timed out after 90.000s.
Expected
    <string>: Summary
to match fields: {
.Node.SystemContainers[pods].Memory:
	Expected
	    <string>: MemoryStats
	to match fields: {
	.PageFaults:
		Expected
		    <uint64>: 1409707
		to be <=
		    <int>: 1000000
	}
	
}

_output/local/go/src/k8s.io/kubernetes/test/e2e_node/summary_test.go:347
```

#### Anything else we need to know:
- links to go.k8s.io/triage appreciated
- links to specific failures in spyglass appreciated

#### Relevant SIG

/sig 

/sig node

**Comment by bobbypage** [comment]
On cgroupv2, it looks like like the number of page faults for `kubepods` cgroup is sometimes higher than is currently expected in the test.

We currently expect this

https://github.com/kubernetes/kubernetes/blob/ce4958d908172117dd011121cd414cc800be9445/test/e2e_node/summary_test.go#L131

however, in a few runs of the test we've seen

`1409707` - https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-cos-cgroupv2-containerd-node-e2e/1441678429651996672
`1268927` - https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-cos-cgroupv2-containerd-node-e2e/1442403211964059648

We may need to adjust this value similar as was done in https://github.com/kubernetes/kubernetes/pull/102726

**Comment by giuseppe** [comment]
@bobbypage yes, good idea.  I think we need to adjust the PageFaults test as well.

I've opened a PR: https://github.com/kubernetes/kubernetes/pull/105313
